tests/Graph/ConstantWeightShortestPathFinder.cpp

```diff
@@ -145,11 +145,11 @@ struct MockGraphDatabase {
 
   ~MockGraphDatabase() {
     for (auto& q : queries) {
-      q->trx()->abort();
+      if (q->trx() != nullptr)
+        q->trx()->abort();
```

Comment:
```suggestion
        q->trx()->abort();
      }
```

we try to avoid:

if (condition) 
  goto fail;
  goto fail;

situations ;)

Reply:
I force-pushed the change so that there's only one commit for this simple fix